src/Diagnostics/FxCop/CSharp/Design/CodeFixes/CA1052CSharpCodeFixProvider.cs

```diff
@@ -39,16 +36,37 @@ public sealed override FixAllProvider GetFixAllProvider()
             if (classDeclaration != null)
```

Comment:
ThrowIfCancellationRequested is fine - the engine catches OperationCanceledExceptions with this cancellation token and handles them properly.